src/modules/function/declaration.rs

```diff
@@ -271,13 +277,74 @@ impl TranslateModule for FunctionDeclaration {
 impl DocumentationModule for FunctionDeclaration {
     fn document(&self, meta: &ParserMetadata) -> String {
         let mut result = vec![];
-        result.push(format!("## `{}`", self.name));
+        result.push(format!("## `{}`\n", self.name));
+
+        let references = self.create_reference(meta, &mut result);
+
         result.push("```ab".to_string());
         result.push(self.doc_signature.to_owned().unwrap());
         result.push("```\n".to_string());
         if let Some(comment) = &self.comment {
             result.push(comment.document(meta));
         }
+
+        if let Some(references) = references {
+            for reference in references {
+                result.push(reference);
+            }
+            result.push("\n".to_string());
+        }
+
         result.join("\n")
     }
 }
+
+impl FunctionDeclaration {
+    #[cfg(debug_assertions)]
+    fn create_reference(&self, meta: &ParserMetadata, result: &mut Vec<String>) -> Option<Vec<String>> {
+        let mut references = Vec::new();
+        let exe_path = env::current_exe()
+            .expect("Executable path not found");
+        let root_path = exe_path.parent()
+            .and_then(Path::parent)
+            .and_then(Path::parent)
+            .expect("Root path not found");
+        let test_path = root_path.join("src")
+            .join("tests")
+            .join("stdlib");
+        let lib_name = meta.context.path.as_ref()
+            .map(Path::new)
+            .and_then(Path::file_name)
+            .and_then(OsStr::to_str)
+            .map(String::from)
+            .unwrap_or_default();
+        result.push(String::from("```ab"));
```

Comment:
Triple backticks are put on their own lines, so the output is consistent with other code blocks.